### ISSUE TITLE
The Future

### ISSUE THREAD

**Issue by SameSkeletonDifferentCloset** [mdotstrange/NightmarePuppeteerPublic#300]
With all that one can do with NP and at its very low price, how about adding more scenes, props, character models, etc. 

You could make a YouTube video and or a news post on Steam whenever you update. Maybe a once a month medium sized update or something adding new content to play with would show even more what one can do. 

If it's as cheap as a stocking stuffer make it so it's refilled. Stockings are where the real presents and surprises are found. NP is the gift that keeps on giving and approaching it in that way could possibly be what makes a big thing come in a little box.

Of course it's up to you and you have other projects but as a user and abuser of NP I think it's a good strategy. 

Thank you.

**Comment by mdotstrange** [comment]
Yeah that is the plan- to add new sets/models/props etc- every month or two. And yes, its overdue for a Steam update post as I've added a lot of stuff that hasn't been documented there.

I'd guess the speed of updates etc will pick up in about 2 months time but I will be looking at the NP code this weekend to try and resolve some of the recent reported issues with Workshop etc.

I think the NP phone I plan to add will bring a lot of new energy to NP so I want to do that before adding a lot of new models etc.
